rocks-db/src/lib.rs

```diff
@@ -383,9 +373,6 @@ impl Storage {
             Self::new_cf_descriptor::<TokenAccountOwnerIdx>(migration_state),
             Self::new_cf_descriptor::<TokenAccountMintOwnerIdx>(migration_state),
             Self::new_cf_descriptor::<MigrationVersions>(migration_state),
-            Self::new_cf_descriptor::<BatchMintToVerify>(migration_state),
```

Comment:
You can not just remove the df_descriptor. You either keep it here, or remove those columns manually from every database (column_remover at your service), or reinvent a migration that removes the stale column families.